driver/vhci/vhci_read.c

```diff
@@ -79,30 +82,44 @@ store_urb_reset_dev(PIRP irp, struct urb_req *urbr)
 static NTSTATUS
 store_urb_reset_pipe(PIRP irp, PURB urb, struct urb_req *urbr)
 {
+	/* 1. We clear STALL/HALT feature on endpoint specified by pipe
```

Comment:
This function can be problematic - It works with USBIP server under linux, but USBIP server under windows handle USB_REQUEST_RESET_PIPE explicitly.